content/2_Bronze/Intro_DS.mdx

```diff
@@ -32,10 +32,9 @@ This creates a `vector` structure that only stores objects of type `string`.
 For our examples below, we will primarily use the `int` data type, but note that
 you can use any data type including `string` and user-defined structures.
 
-Essentially, every standard library data structure supports the `size()` method,
-which returns the number of elements in the data structure, and the `empty()`
-method, which returns `true` if the data structure is empty, and `false`
-otherwise.
+Every standard library data structure supports the `size()` method, which
```

Comment:
This isn't true, things like `std::tuple` and `std::pair` don't. Clarify this

Reply:
dustin if you mean you can't get their size,
1. https://www.cplusplus.com/reference/tuple/tuple_size/
2. if you're telling me you don't know how many elements are in a _pair_...